src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.wasm.cs

```diff
@@ -187,5 +192,29 @@ public int SelectionLength
 				_textBoxView.SelectionEnd = _textBoxView.SelectionStart + value;
 			}
 		}
+
+		partial void OnEnterKeyHintChangedPartial(EnterKeyHint enterKeyHint)
+		{
+			_enterKeyHint = enterKeyHint;
+			ApplyEnterKeyHint();
+		}
+
+		private void ApplyEnterKeyHint()
+		{
+			if (_enterKeyHint is not { } enterKeyHint ||
+				TextBoxView is not { } textBoxView)
+			{
+				return;
+			}
+
+			var enterKeyHintValue = enterKeyHint switch
+			{
+				EnterKeyHint.Default => "",
+				_ when Enum.IsDefined(typeof(EnterKeyHint), enterKeyHint) => enterKeyHint.ToString("g").ToLowerInvariant(),
```

Comment:
Why is the "g" required in the `ToString("g")`?

Reply:
Indeed, not needed!